server/gorm/deletions.go

```diff
@@ -71,6 +71,7 @@ func (c *Client) DeleteChildrenOfProject(ctx context.Context, project names.Proj
 // DeleteChildrenOfApi deletes all the children of a api.
 func (c *Client) DeleteChildrenOfApi(ctx context.Context, api names.Api) error {
 	for _, entityName := range []string{
+		storage.ArtifactEntityName,
 		models.BlobEntityName,
```

Comment:
aside: it seems odd that we didn't also move this name into the "storage" package, not urgent

Reply:
Just noticed I missed this comment after checking my email. I agree, I missed this one when I moved the others over.

#147 seems like it will cover this.

Reply:
SGTM